# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1319682807 14400
#      Wed Oct 26 22:33:27 2011 -0400
# Node ID 9586dc4e838b034077e2a08b70752f2194c2791c
# Parent  1e81e2e30af33f619976dc8b8af49417bba860b1
double the size of '.' markers produced by the OpenGL renderer

* gl-render.cc (opengl_renderer::init_marker):
Double size of '.' markers.

diff --git a/src/gl-render.cc b/src/gl-render.cc
--- a/src/gl-render.cc
+++ b/src/gl-render.cc
@@ -2882,17 +2882,17 @@ opengl_renderer::make_marker_list (const
       glEnd ();
       break;
     case '.':
       {
         double ang_step = M_PI / 5;
 
         glBegin (GL_POLYGON);
         for (double ang = 0; ang < (2*M_PI); ang += ang_step)
-          glVertex2d (sz*cos(ang)/6, sz*sin(ang)/6);
+          glVertex2d (sz*cos(ang)/3, sz*sin(ang)/3);
         glEnd ();
       }
       break;
     case 's':
       glBegin ((filled ? GL_POLYGON : GL_LINE_LOOP));
       glVertex2d (-sz/2, -sz/2);
       glVertex2d (-sz/2, sz/2);
       glVertex2d (sz/2, sz/2);
